tests/multi_bluetooth/ble_gap_advertise.py

```diff
@@ -51,12 +54,12 @@ def irq(ev, data):
     except:
         pass
     ble.irq(irq)
-    ble.gap_scan(2 * ADV_TIME_S * 1000, 10000, 10000)
+    ble.gap_scan(2 * ADV_TIME_MS, 30000, 30000)
```

Comment:
Was it intentional to change the 2nd and 3rd arguments here?

Reply:
Yes, 30000 is the minimum.